app/assets/stylesheets/govuk_publishing_components/components/_reorderable-list.scss

```
@@ -0,0 +1,120 @@
+.gem-c-reorderable-list {
+  @include govuk-font(19, bold);
+
+  list-style-type: none;
+  margin-bottom: govuk-spacing(6);
+  margin-top: 0;
+  padding-left: 0;
+  position: relative;
+
+  .govuk-form-group {
+    margin-bottom: 0;
+  }
+}
+
+.gem-c-reorderable-list__item {
+  margin-bottom: govuk-spacing(3);
+  border: 1px solid $govuk-border-colour;
+  padding: govuk-spacing(3);
+}
+
+.gem-c-reorderable-list__item--chosen {
+  background-color: govuk-colour('light-grey');
+  outline: 2px dotted $govuk-border-colour;
+}
+
+.gem-c-reorderable-list__item--drag {
+  background-color: govuk-colour('white');
+  list-style-type: none;
+
+  .gem-c-reorderable-list__actions {
+    visibility: hidden;
+  }
+}
+
+.gem-c-reorderable-list__wrapper {
+  display: block;
+
+  @include govuk-media-query($from: desktop) {
+    display: inline-flex;
+    width: 100%;
+  }
+}
+
+.gem-c-reorderable-list__content,
+.gem-c-reorderable-list__actions {
+  margin-bottom: govuk-spacing(2);
+}
+
+.gem-c-reorderable-list__content {
+  @include govuk-media-query($from: desktop) {
```

This suggestion is for content to have some space on small screens, when actions are placed under content.

```suggestion
  margin-bottom: govuk-spacing(2);
  @include govuk-media-query($from: desktop) {
    margin-bottom: 0;
```